go/billing/tasks.py

```diff
@@ -105,6 +105,27 @@ def get_session_transactions(transactions):
     return transactions
 
 
+def get_session_length_transactions(transactions):
+    transactions = transactions.filter(
+        transaction_type=Transaction.TRANSACTION_TYPE_MESSAGE)
+
+    transactions = transactions.values(
+        'tag_pool_name',
+        'tag_name',
+        'provider',
+        'session_unit_time',
+        'session_length_cost',
```

Comment:
We should group on ``session_unit_cost`` here instead of ``session_length_cost`` (same issue as one described in comment in test for this).